lib/Extension/LanguageServerPhpCsFixer/Tests/VersionResolver/PhpCsFixerVersionResolverTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerPhpCsFixer\Tests\VersionResolver;
+
+use PHPUnit\Framework\TestCase;
+use Phpactor\Extension\LanguageServerPhpCsFixer\VersionResolver\PhpCsFixerVersionResolver;
+use Psr\Log\NullLogger;
+
+use function Amp\call;
+use function Amp\Promise\wait;
+
+class PhpCsFixerVersionResolverTest extends TestCase
+{
+    public function testResolve(): void
+    {
+        $path = realpath(__DIR__ . '/../../../../../vendor/bin/php-cs-fixer');
+
+        self::assertNotFalse($path);
+
+        $resolver = new PhpCsFixerVersionResolver($path, new NullLogger());
+
+        $process = call(function () use ($resolver) {
+            $version = yield $resolver->resolve();
+
+            self::assertSame('3.89.1', $version->__toString());
```

Comment:
probably just asserting that it's a valid semantic version would be less of a headache, maybe just match `{^[0-9]+.[0-9]+.[0-9+].*$` ?